src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -922,16 +949,20 @@ public void destroy() throws Exception {
 
     private static class NoOpEventPublisher implements EventPublisher {
         @Override
-        public void publish(Object o) {}
+        public void publish(Object o) {
```

Comment:
don't reformat unrelated code as this makes the review harder and make sure to run `spotless:apply`